packages/hardhat/test/e2e/periphery/controller-helper.ts

```diff
@@ -352,10 +363,211 @@ describe("ControllerHelper: mainnet fork", function () {
+      // console.log('vaultBefore.shortAmount', vaultBefore.shortAmount.toString())
+      // console.log('vaultBefore.NftCollateralId', vaultBefore.NftCollateralId.toString())
+      // console.log('vaultAfter.collateralAmount', vaultAfter.collateralAmount.toString())
+      // console.log('vaultAfter.shortAmount', vaultAfter.shortAmount.toString())
+      // console.log('vaultAfter.NftCollateralId', vaultAfter.NftCollateralId.toString())
+      // console.log('mintWSqueethAmount', mintWSqueethAmount.toString())
+      // console.log('collateralToMint', collateralToMint.toString())
+      // console.log('collateralToFlashloan', collateralToFlashloan.toString())
+      // console.log('gasSpent', gasSpent.toString())
+      // console.log('ownerOfUniNFT', ownerOfUniNFT.toString())
+      // console.log('controller', controller.address.toString())
+      // console.log('depositorEthBalanceBefore', depositorEthBalanceBefore.toString())
+      // console.log('depositorEthBalanceAfter', depositorEthBalanceAfter.toString())
+
+      expect(BigNumber.from(vaultAfter.NftCollateralId).eq(tokenId)).to.be.true;
+      expect(position.tickLower === -887220).to.be.true
+      expect(position.tickUpper === 887220).to.be.true
+      expect(ownerOfUniNFT === controller.address).to.be.true
+      //expect(vaultAfter.shortAmount.sub(vaultBefore.shortAmount).sub(mintWSqueethAmount).abs().lte(1000)).to.be.true
+      expect(vaultAfter.collateralAmount.sub(vaultBefore.collateralAmount).sub((collateralToMint.sub(collateralToFlashloan))).eq(0)).to.be.true
+      expect(depositorEthBalanceAfter.sub(depositorEthBalanceBefore).sub(gasSpent).sub((collateralToMint.sub(collateralToFlashloan))).lte(1)).to.be.true
+    })
+
+    it("open short in new vault, mint, LP oSQTH only, deposit LP NFT, no eth added", async () => {
+      // New vault with 4x collateral
+      const normFactor = await controller.getExpectedNormalizationFactor()
+      const mintWSqueethAmount : BigNumber = ethers.utils.parseUnits('40')
+      const mintRSqueethAmount = mintWSqueethAmount.mul(normFactor).div(one)
+      const ethPrice = await oracle.getTwap(ethUsdcPool.address, weth.address, usdc.address, 420, true)
+      const scaledEthPrice = ethPrice.div(10000)
+      const debtInEth = mintRSqueethAmount.mul(scaledEthPrice).div(one)
+      const collateralToDeposit = debtInEth.mul(4)
+      await controller.connect(depositor).mintWPowerPerpAmount(0, mintWSqueethAmount, 0, {value: collateralToDeposit})
+      const vaultId = (await shortSqueeth.nextId()).sub(1);
+      await controller.connect(depositor).updateOperator(vaultId, controllerHelper.address)
+      // Get before context
+      const vaultBefore = await controller.vaults(vaultId)
+      const depositorEthBalanceBefore = await ethers.provider.getBalance(depositor.address)
+
+      // Set up for full range LP with half collateral flashloaned
+      const flashloanWMintDepositNftParams = {
```

Comment:
Shouldn't this tick lower and upper be based on if weth is token0 or not, similar to Leone test, just to make sure it never fail in CI...etc ?

Reply:
done - now going to [-887220 --> below current tick] or [above curent tick --> 887220]